docs/project/2024/Ruxandra.popa/index.md

```diff
@@ -0,0 +1,115 @@
+| [0.96'' I2C IIC OLED Display](https://www.sigmanortec.ro/Display-OLED-0-96-I2C-IIC-Albastru-p135055705?gad_source=1&gclid=EAIaIQobChMI87uZ2uzuhQMVXAUGAB0WEwaqEAQYASABEgIC8vD_BwE) | User interface display       | [20 RON](https://www.sigmanortec.ro/Display-OLED-0-96-I2C-IIC-Albastru-p135055705?gad_source=1&gclid=EAIaIQobChMI87uZ2uzuhQMVXAUGAB0WEwaqEAQYASABEgIC8vD_BwE) |
+| [VISATON 2916](https://ro.farnell.com/visaton/2916/speaker-k-50-wp-50-ohms/dp/1683896?gross_price=true&CMP=KNC-GRO-GEN-SHOPPING-PMax_Test_840_HighMargin&mckv=_dc%7Cpcrid%7C%7Cplid%7C%7Ckword%7C%7Cmatch%7C%7Cslid%7C%7Cproduct%7C1683896%7Cpgrid%7C%7Cptaid%7C%7C&gad_source=1&gclid=EAIaIQobChMIqpmFqOzuhQMVdkJBAh25UwYvEAQYASABEgJ7jPD_BwE)   | Audio output device          | [50 RON](https://ro.farnell.com/visaton/2916/speaker-k-50-wp-50-ohms/dp/1683896?gross_price=true&CMP=KNC-GRO-GEN-SHOPPING-PMax_Test_840_HighMargin&mckv=_dc%7Cpcrid%7C%7Cplid%7C%7Ckword%7C%7Cmatch%7C%7Cslid%7C%7Cproduct%7C1683896%7Cpgrid%7C%7Cptaid%7C%7C&gad_source=1&gclid=EAIaIQobChMIqpmFqOzuhQMVdkJBAh25UwYvEAQYASABEgJ7jPD_BwE) |
+| [Card MicroSD](https://www.emag.ro/card-de-memorie-microsd-kingston-canvas-select-plus-32gb-100mb-s-cu-adaptor-sdcs2-32gb/pd/D63FBGBBM/?ref=fam#32-GB) | Memory Card MicroSD | [30-40 RON](https://www.emag.ro/card-de-memorie-microsd-kingston-canvas-select-plus-32gb-100mb-s-cu-adaptor-sdcs2-32gb/pd/D63FBGBBM/?ref=fam#32-GB) |
+| [Electronic components: LEDs, buttons, wires resistors and Breadboard](https://www.optimusdigital.ro/en/?gad_source=1&gclid=EAIaIQobChMIt8fHl_DuhQMVoj8GAB2CHwcJEAAYASAAEgLBrvD_BwE) | Electronic Components | [15-20 RON](https://www.optimusdigital.ro/en/?gad_source=1&gclid=EAIaIQobChMIt8fHl_DuhQMVoj8GAB2CHwcJEAAYASAAEgLBrvD_BwE) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+| [TinyUSB](https://github.com/hathach/tinyusb) | An open-source USB library for embedded systems that handles USB device and host functionality | Provides USB communication capabilities for the RustPlayer, enabling potential future expansions such as connecting USB-based audio devices |
+| [FatFs](https://github.com/abbrev/fatfs) | A lightweight file system library for embedded systems, compatible with FAT32, FAT16, and FAT12 | Handles reading and writing audio files to and from the microSD card efficiently |
+| [ESP8266Audio](https://github.com/earlephilhower/ESP8266Audio) | This library manages the I2S digital audio interface for Raspberry Pi Pico W, ensuring high-quality audio playback | Configures and controls the PCM5102 DAC for superior audio output from the MP3 files played |
```

Comment:
I think this lubrary is for the ESP8266 device which us not ARM.

Reply:
I have modified the libraries, thank you!